WordPress/src/main/java/org/wordpress/android/ui/jetpack/common/ViewType.kt

```diff
@@ -8,5 +8,7 @@ enum class ViewType(val id: Int) {
     THREATS_HEADER(4),
     THREAT_ITEM(5),
     CHECKBOX(6),
-    BACKUP_SUB_HEADER(7)
+    BACKUP_SUB_HEADER(7),
+    THREAT_CONTEXT_LINES(8),
+    FILE_NAME(9)
```

Comment:
@ashiagr - Is FILE_NAME is exclusive to the scan feature and if so, should we prefix it as such like we did for the others (backup, threats)? wdyt?

Reply:
Good point. Renamed in aa33ca5.